versioned_docs/version-25.10/alerts-notifications/ticketing.md

```diff
@@ -7,6 +7,8 @@ import TabItem from '@theme/TabItem';
 
 ## Configuration
 
+> Warning! There is a chance of your custom configuration disappearing when updating Centreon. To avoid this issue, refer to our [Update Centreon](../update/update-centreon-platform.md#Perform-a-backup) procedure.
```

Comment:
```suggestion
> Warning! Open Ticket users with a custom configuration should [perform a backup before updating Centreon](../update/update-centreon-platform.md#Perform-a-backup) to avoid them being erased during the update process.
```